lib/manager/npm/extract/monorepo.ts

```diff
@@ -13,10 +22,115 @@ function matchesAnyPattern(val: string, patterns: string[]): boolean {
   return res;
 }
 
-export function detectMonorepos(
+export async function extractPnpmFilters(
```

Comment:
move the pnpm function to `./pnpm.ts` so it's easier to tes them.

Reply:
changed in 28a36aa130334e844a89884a6cb31c3fbf528659